MFVNeutralino/test/TrackMover/mctruth_output.out

```diff
@@ -0,0 +1,68 @@
+./mctruth.exe with options: in_fn: /eos/uscms/store/group/lpclonglived/pkotamni/WplusH_HToSSTodddd_WToLNu_MH-125_MS-55_ctauS-1_TuneCP5_13TeV-powheg-pythia8/TrackMoverMCTruthOnnormdzULV30LepMumv6_2017/240226_125240/0000/mctruth_2.root out_fn: hists.root tree_path: mfvMovedTreeMCTruth/t json: none every/quiet/silent: 0/0/0 num_chunks: 1 which_chunk: 0 weights: 1 pu_weights: none half mc by lumi: 0 quarter mc by lumi: 0
```

Comment:
We don;t need to track this mctruth_output.out file, right? I think it can be dropped from the repo.